swarm/network/simulation/kademlia.go

```diff
@@ -94,3 +96,53 @@ func (s *Simulation) kademlias() (ks map[enode.ID]*network.Kademlia) {
 	}
 	return ks
 }
+
+func (s *Simulation) WaitNetworkHealth() error {
+	// construct the peer pot, so that kademlia health can be checked
+	addrs := make([][]byte, len(s.Net.Nodes))
+	ids := make([]enode.ID, len(s.Net.Nodes))
+	for i, v := range s.Net.Nodes {
+		ids[i] = v.ID()
+		addrs[i] = v.ID().Bytes()
+	}
+	// construct the peer pot, so that kademlia health can be checked
+	ppmap := network.NewPeerPotMap(2, addrs) //yes i'm hardcoding this
```

Comment:
In the kademlia refactor series the `MinProxBinSize` parameter is stored in the simulation upon instantiation, and not (ludicrously) passed with this call

Reply:
i know. therefore the cheeky act and comment

Reply:
?

Reply:
@zelig this is to be changed with the minProxBinSize change PR. not sure what i should write here for now?